src/NuGetPackageVerifier/Rules/PackageOwnershipRule.cs

```diff
@@ -25,7 +25,6 @@ public class PackageOwnershipRule : IPackageVerifierRule
             "microsoft",
         };
         private static readonly IList<string> OwnedPackages = GetOwnedPackages();
-        private readonly HttpClient _httpClient = new HttpClient();
```

Comment:
This can just be `static`. Having a static `HttpClient` should be fine as long as we don't store any http response.